prepare.php

```diff
@@ -15,6 +15,7 @@
 $WPT_SSH_OPTIONS = getenv( 'WPT_SSH_OPTIONS' ) ? : '-o StrictHostKeyChecking=no';
 $WPT_TEST_DIR = getenv( 'WPT_TEST_DIR' );
 $WPT_PHP_EXECUTABLE = getenv( 'WPT_PHP_EXECUTABLE') ? : 'php';
+$WPT_DEBUG = getenv( 'WPT_DEBUG') ? : '';
```

Comment:
Would there be an issue with having this get set to false if it isn't assigned, rather than setting an empty string?

Reply:
Ah, nit-pick, but looks like this may also need a space after `'WPT_DEBUG'` before the parenthesis.